crates/anonymization/src/core/hash.rs

```diff
@@ -75,7 +90,7 @@ impl Hasher {
                 }
                 hasher.update(data);
 
-                Ok(general_purpose::STANDARD.encode(hasher.finalize()))
+                Ok(hasher.finalize().to_vec())
```

Comment:
Having to clone the constant size hash value into a variable size vector value is sad. Can't you return the array directly? All methods should give you a 256-bit value.